airbyte-integrations/connectors/source-paystack/pyproject.toml

```diff
@@ -3,26 +3,25 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.1.4"
+version = "0.1.5"
 name = "source-paystack"
-description = "Source implementation for Paystack."
+description = "Source implementation for paystack."
 authors = [ "Airbyte <contact@airbyte.io>",]
 license = "MIT"
 readme = "README.md"
 documentation = "https://docs.airbyte.com/integrations/sources/paystack"
 homepage = "https://airbyte.com"
 repository = "https://github.com/airbytehq/airbyte"
-[[tool.poetry.packages]]
-include = "source_paystack"
+packages = [ { include = "source_paystack" }, {include = "main.py" } ]
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "0.80.0"
+airbyte-cdk = "^1"
 
 [tool.poetry.scripts]
 source-paystack = "source_paystack.run:run"
 
 [tool.poetry.group.dev.dependencies]
-requests-mock = "^1.12.1"
-pytest = "^6.1"
-pytest-mock = "^3.6.1"
+requests-mock = "*"
+pytest-mock = "*"
```

Comment:
In this line, for some reason `-mock` is highlighted — are you sure there are no funky utf-related diffs?